pkg/container-disk/container-disk_test.go

```diff
@@ -182,8 +201,13 @@ var _ = Describe("ContainerDisk", func() {
 					},
 				})
 
-				vmi := api.NewMinimalVMI("fake-vmi")
-				appendContainerDisk(vmi, "r0")
+				vmi := libvmi.New(
+					libvmi.WithContainerDisk("r0", "someimage:v1.2.3.4"),
+					libvmi.WithResourceMemory("64M"),
+					libvmi.WithResourceCPU("1"),
```

Comment:
After adding this, are the lines below still needed?

Reply:
which lines  ?

Reply:
yes , i already removed these lines ( if you mean these line )
```go
By("Setting the VMI's resource requirements")
vmi.Spec.Domain.Resources = v1.ResourceRequirements{
Requests: k8sv1.ResourceList{
k8sv1.ResourceCPU:    resource.MustParse("1"),
k8sv1.ResourceMemory: resource.MustParse("64M"),
},
Limits: k8sv1.ResourceList{
k8sv1.ResourceCPU:    resource.MustParse("1"),
k8sv1.ResourceMemory: resource.MustParse("64M"),
},
}
```
and I got this error 
```bash
meline >>
  {"component":"","level":"info","msg":"Updating cluster config from KubeVirt to resource version 'm66xmkq9mm'","pos":"configuration.go:365","timestamp":"2024-06-14T11:50:16.416341Z"}
  [FAILED] in [It] - /home/ali/Projects/kubevirt/pkg/container-disk/container-disk_test.go:201 @ 06/14/24 14:50:16.416
  << Timeline

  [FAILED] Expected
      <v1.ResourceList | len:3>: {
          "ephemeral-storage": {
              i: {value: 50, scale: 6},
              d: {Dec: nil},
              s: "50M",
              Format: "DecimalSI",
          },
          "cpu": {
              i: {value: 1, scale: -3},
              d: {Dec: nil},
              s: "1m",
              Format: "DecimalSI",
          },
          "memory": {
              i: {value: 1, scale: 6},
              d: {Dec: nil},
              s: "1M",
              Format: "DecimalSI",
          },
      }
  to contain elements
      <[]resource.Quantity | len:3, cap:3>: [
          {
              i: {value: 10, scale: -3},
              d: {Dec: nil},
              s: "10m",
              Format: "DecimalSI",
          },
          {
              i: {value: 40, scale: 6},
              d: {Dec: nil},
              s: "40M",
              Format: "DecimalSI",
          },
          {
              i: {value: 50, scale: 6},
              d: {Dec: nil},
              s: "50M",
              Format: "DecimalSI",
          },
      ]
  the missing elements were
      <[]resource.Quantity | len:2, cap:2>: [
          {
              i: {value: 10, scale: -3},
              d: {Dec: nil},
              s: "10m",
              Format: "DecimalSI",
          },
          {
              i: {value: 40, scale: 6},
              d: {Dec: nil},
              s: "40M",
              Format: "DecimalSI",
          },
      ]

```